tests/test_tasks.py

```diff
@@ -243,6 +244,66 @@ def test_filter_manifest_keywords_syntax_error():
     assert return_value == 3
 
 
+@pytest.mark.parametrize(
+    "tags,session_count",
```

Comment:
are tags case sensitive? I wonder if adding a case here for that would make sense?

```
        (["Foo"], 0),
```

similarly, most of the tags match function names. if we have the tag `foo_bar_baz` it should also match 0

Reply:
@crwilcox thanks for the suggestions, I added test cases for tags not found in `test_filter_manifest_tags_not_found`. I also just changed the function names so make sure we're matching tags and not session names. Let me know what you think.